packages/@o3r/analytics/project.json

```diff
@@ -112,6 +112,16 @@
       "options": {
         "script": "build:builders"
       }
+    },
+    "documentation": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "compodoc"
+      },
+      "outputs": [
+        "{workspaceRoot}/generated-doc/@o3r/analytics/**/*"
```

Comment:
May it can also be defined in `nx.json` with `{workspaceRoot}/generated-doc/{projectName}/**/*` (but the projectName is not the packageName)